Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -168,12 +168,11 @@ open class ChatChannelListVC: _ViewController,
             withReuseIdentifier: collectionViewCellReuseIdentifier,
             for: indexPath
         ) as! ChatChannelListCollectionViewCell
-    
+
+        guard let channel = getChannel(at: indexPath) else { return UICollectionViewCell() }
```

Comment:
Returning empty cell like this will crash the app too (due to unregistered cell identifier).
Could you log this case?
Thanks

Reply:
I wrapped the logging & assertion logic in getChannel